fdbserver/storageserver.actor.cpp

```diff
@@ -6866,14 +6881,40 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 	// delay(0) to force a return to the run loop before the work of fetchKeys is started.
 	//  This allows adding->start() to be called inline with CSK.
 	try {
+		if (conductBulkLoad) {
+			TraceEvent(SevDebug, "BulkLoadFetchKeysBeforeCoreStarted", data->thisServerID)
```

Comment:
Probably adding "SS" to the prefix which helps you to quickly find which role triggers the event.